src/Nethermind/Chains/op-sepolia.json

```diff
@@ -62,6 +63,7 @@
     "eip5656TransitionTimestamp": "0x65D62C10",
     "eip6780TransitionTimestamp": "0x65D62C10",
     "rip7212TransitionTimestamp": "0x66575100",
+    "OpGraniteTransitionTimestamp": "0x66ba3180",
```

Comment:
could you make it `opGraniteTransitionTimestamp`

Reply:
Done